src/psyclone/domain/gocean/transformations/gocean_loop_fuse_trans.py

```diff
@@ -84,13 +83,8 @@ def validate(self, node1, node2, options=None):
         :raises TransformationError: if invalid parameters are passed in.
 
         '''
-
-        # Either both nodes are gocean1.0 loop nodes, or both
-        # nodes are gocean0.1 loop nodes, otherwise raise an exception:
-        if not ((isinstance(node1, psyclone.gocean0p1.GOLoop) and
-                 isinstance(node2, psyclone.gocean0p1.GOLoop)) or
-                (isinstance(node1, psyclone.gocean1p0.GOLoop) and
-                 isinstance(node2, psyclone.gocean1p0.GOLoop))):
+        if not (isinstance(node1, psyclone.gocean1p0.GOLoop) and
+                isinstance(node2, psyclone.gocean1p0.GOLoop)):
             raise TransformationError("Error in {0} transformation. "
```

Comment:
Since we're here, please could you change this to use an f-string.

Reply:
Aha, yes, see previous comment :)